09-configuration_management_policy.md

```diff
@@ -18,8 +18,8 @@ Health Note documents of all changes to production systems and networks.
 
 ## 9.2 Configuration Management Policies
 
-1. No systems are deployed into Health Note environments without approval of the Health Note CTO.
-2. All changes to production systems, network devices, and firewalls are approved by the Health Note CTO before they are implemented to assure they comply with business and security requirements.
+1. No systems are deployed into Health Note environments without approval of the Health Note CTO or Head of Engineering.
+2. All changes to production systems, network devices, and firewalls are approved by the Health Note CTO or Head of Engineering before they are implemented to assure they comply with business and security requirements.
 3. All changes to production systems are tested before they are implemented in production.
 4. Implementation of approved changes are only performed by authorized personnel.
 5. An up-to-date inventory of systems is maintained using Google spreadsheets and architecture diagrams hosted on Google Apps. All systems are categorized as production and utility to differentiate based on criticality.
```

Comment:
Do we have this?

Reply:
For 9 below, do we need that? I'm not sure what it's solving for

Reply:
We have an informal process for approval of changes that isn't well documented.  Details of the PR/peer review process for changes should be added.

NTP/time syncing is required so that authentication can't be tampered with as it affects SSL certificates, 2FA, etc.